src/antlr/SqlBaseParser.ts

```diff
@@ -1248,10 +1248,7 @@ export class SqlBaseParser extends Parser {
     return SqlBaseParser._serializedATN;
   }
 
-  protected createFailedPredicateException(
```

Comment:
This file (`SqlBaseParser.ts`) is fully generated from `SqlBase.g4`. Shall we avoid fixing it because when grammar is changed it will be regenerated.

Reply:
In other places, we have applied fixes / formatting as part of generation. We could do the same here, too.

Advantages:

* If you ever look at the generated files, they are nicely formatted / easy to browse and understand
* If new warnings / errors appear, something might be wrong in the generated file / logic for code generation

Reply:
Ok